### PR TITLE
Use IntegateEPP in ComputeCalibrationCoefVan

### LINKED ISSUE #20472: Update ComputeCalibrationCoefVan to use IntegrateEPP
[`ComputeCalibrationCoefVan`](http://docs.mantidproject.org/nightly/algorithms/ComputeCalibrationCoefVan-v1.html) currently integrates the vanadium workspace manually. It should rather use the [`IntegrateEPP`](http://docs.mantidproject.org/nightly/algorithms/IntegrateEPP-v1.html) algorithm as its backend.

### PR DESCRIPTION
**Pull request by soininen**
[`ComputeCalibrationCoefVan`](http://docs.mantidproject.org/nightly/algorithms/ComputeCalibrationCoefVan-v1.html) was doing manual summation when integrating the reference (vanadium) workspace. This PR changes the algorithm to use [`IntegrateEPP`](http://docs.mantidproject.org/nightly/algorithms/IntegrateEPP-v1.html) instead, thus standardizing the integration method (`IntegrateEPP` uses `Integration`) and improving performance. `IntegrateEPP` also handles the integration limits in a more precise manner than what was done in `ComputeCalibrationCoefVan` before.

**To test:**

Code review. One could run the following script before and after applying these changes to check the `norm_coefs` are roughly the same (they won't be exactly because the different integration range handling).
```python
# The next line needs user intervention.
ws = Load('<mantid_build_directory>/ExternalData/Testing/Data/UnitTest/ILL/IN4/084446.nxs')

epps = FindEPP(ws)

ComputeCalibrationCoefVan(VanadiumWorkspace=ws, OutputWorkspace='norm_coefs', Temperature=1011.0, EPPTable=epps)
```

Close #20472.

**Release Notes** 

*Direct inelastic spectrometry release notes were updated accordingly.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by samueljackson92 — CHANGES_REQUESTED** [review]
One minor comment. Otherwise this looks good to me.

File: Framework/PythonInterface/test/python/plugins/algorithms/ComputeCalibrationCoefVanTest.py [hunk 1]
```diff
@@ -53,10 +53,14 @@ def test_sum(self):
         wsoutput = AnalysisDataService.retrieve(outputWorkspaceName)
 
         # check whether sum is calculated correctly, for theta=0, dwf=1
-        y_sum = sum(self._input_ws.readY(0)[27:75])
-        e_sum = np.sqrt(sum(np.square(self._input_ws.readE(0)[27:75])))
-        self.assertAlmostEqual(y_sum, wsoutput.readY(0)[0])
-        self.assertAlmostEqual(e_sum, wsoutput.readE(0)[0])
+        y_sumMin = np.sum(self._input_ws.readY(0)[28:73])
```
[the remark this answers is not in the document]
Comment (soininen) [reply Framework/PythonInterface/test/python/plugins/algorithms/ComputeCalibrationCoefVanTest.py]: The test data is summed over the Gaussian peak (+/- 3*hwhm) by the algorithm. These magic numbers correspond to that integration range. I just adjusted the tests to comply with the slightly changed integration ranges: originally, the algorithm included partial bins at the ends of the range and added a single bin 'too far'. With `IntegrateEPP`, the summation range is more precise as partial bins are handled correctly.

**Comment by mganeva** [comment]
The PR produces a coefficients workspace with more reasonable X axis. The TOFTOF data reduction works as before. Can be merged, imho.

File: Framework/PythonInterface/test/python/plugins/algorithms/ComputeCalibrationCoefVanTest.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (samueljackson92) [reply Framework/PythonInterface/test/python/plugins/algorithms/ComputeCalibrationCoefVanTest.py]: Could that be a comment? or use constants set to those values called something that indicate that this is the expected integration range for the test?

**Review by samueljackson92 — APPROVED** [review]

**Comment by samueljackson92** [comment]
Looks good to me. Wait for builds to pass then :shipit:

**Pull request merged**